src/ChipInput.js

```diff
@@ -210,8 +210,11 @@ class ChipInput extends React.Component {
 
   handleInputBlur = (event) => {
     if (!this.autoComplete.refs.menu) {
-      if (!this.props.persistInput) this.setState({ inputValue: '' });
-      this.setState({ isFocused: false });
+      if (this.props.clearOnBlur !== undefined && this.props.clearOnBlur) {
```

Comment:
This would be easier (and safer) by adding `clearOnBlur: true` to the `defaultProps` below.